### ISSUE TITLE
Why do I need to create another subkey when GPG creates a default subkey

### ISSUE THREAD

**Issue by tachang** [StackExchange/blackbox#168]
Why do you need to create another subkey when gpg by default creates a subkey? Couldn't you use that subkey and set an empty password?

Is it because that subkey is encryption only and we need a subkey that supports decryption too? But then why select option 6:

Please select what kind of key you want:
   (3) DSA (sign only)
   (4) RSA (sign only)
   (5) Elgamal (encrypt only)
   (6) RSA (encrypt only)

For a key that is encrypt only?

**Comment by tlimoncelli** [comment]
I confess that the reason for this decision was that "it worked".  I tried a few other things that didn't work and this was the first one that did work.  Maybe I didn't see the default subkey?  (I don't know, I didn't keep notes about that).

I'm not a crypto expert, just a GPG user.  I'd be glad to change the process if you find something that works better.

**Comment by tachang** [comment]
Yeah I just wanted to open up a discussion about it. Maybe someone who uses GPG a lot more and understands the output will chime in.
